maven-scm-plugin/src/test/java/org/apache/maven/scm/plugin/BranchMojoTest.java

```diff
@@ -79,36 +85,33 @@ protected void setUp()
         checkoutMojo.execute();
     }
 
+    @Test
     public void testBranch()
         throws Exception
     {
-        if ( !ScmTestCase.isSystemCmd( SvnScmTestUtils.SVN_COMMAND_LINE ) )
-        {
-            ScmTestCase.printSystemCmdUnavail( SvnScmTestUtils.SVN_COMMAND_LINE, getName() );
-            return;
-        }
+        checkScmPresence( SvnScmTestUtils.SVNADMIN_COMMAND_LINE, "testBranch" );
 
         BranchMojo mojo =
-            (BranchMojo) lookupMojo( "branch", getTestFile( "src/test/resources/mojos/branch/branch.xml" ) );
+            (BranchMojo) lookupMojo( "branch", PlexusJUnit4TestSupport.getTestFile( "src/test/resources/mojos/branch/branch.xml" ) );
```

Comment:
Because this is a very incomplete draft.
My first goal was to make it work an then restructure to make it clean.
I have note been able to reach this first goal yet.

Reply:
So just to be clear. I'm totally fine if you guys pick this one up.
I'm stuck here.